Platform/servicecontrol/servicecontrol-instance/bin/ServiceControl.exe.config

```diff
@@ -7,6 +7,7 @@ These settings are only here so that we can debug ServiceControl while developin
   <appSettings>
     <add key="ServiceControl/HostName" value="localhost" />
     <add key="ServiceControl/Port" value="33533" />
+    <add key="ServiceControl/MaintenancePort" value="33533" />
```

Comment:
Can this be the same port? I guess it doesn't matter because we don't put this instance into maintenance mode?